supplementary/alica_ros1/alica_ros_turtlesim/src/base.cpp

```diff
@@ -25,61 +19,26 @@ namespace turtlesim
 {
 
 Base::Base(ros::NodeHandle& nh, ros::NodeHandle& priv_nh, const std::string& name, const int agent_id, const std::string& roleset,
-        const std::string& master_plan, const std::string& path, bool loadDynamically)
+        const std::string& master_plan, const std::string& path)
         : spinner(0)
-        , _loadDynamically(loadDynamically)
 {
     // Initialize Alica
     ac = new alica::AlicaContext(AlicaContextParams(name, path + "/etc/", roleset, master_plan, false, agent_id));
 
     ac->setCommunicator<alicaRosProxy::AlicaRosCommunication>();
     ac->setTimerFactory<alicaRosTimer::AlicaRosTimerFactory>();
     ac->setLogger<alicaRosLogger::AlicaRosLogger>(agent_id);
```

Comment:
Maybe we should still add world model to the global blackboard rather than in init behavior?

Reply:
Its still added to the global blackboard. But its done in the libalica-turtlesim library to demonstrate how a library should add library specific world models to the blackboard